microscopy_SEM001/sub-01/sub-01_sessions.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I think this file should be named `sub-01_scans.json`  instead of _sessions.json?

Reply:
If this is the case, then it must include a `filename` column

Reply:
Also when describing session_id, it says "sample" ID, it seems to be mixing sessions and samples?